src/components/technologies/genericTechnologies/conv1.py

```diff
@@ -207,7 +234,7 @@ def init_max_input(const, t, car):
 
     def _performance_function_type_1(self, b_tec):
         """
-        Linear, no minimal partload, through origin
+        Linear, through origin, min partload possible
```

Comment:
in case of a ML it doesn't go through the origin right? doesn't matter to me but maybe to keep it precise

Reply:
The line still goes through the origin. But it cannot reach the origin. I left it as it was.